UNITTESTS/stubs/ATHandler_stub.cpp

```diff
@@ -241,14 +210,15 @@ void ATHandler::restore_at_timeout()
 void ATHandler::process_oob()
 {
     if (ATHandler_stub::process_oob_urc) {
-        size_t prefix_len = 0;
-        for (struct oob_t *oob = _oobs; oob; oob = oob->next) {
-            prefix_len = oob->prefix_len;
-            if (!memcmp(oob->prefix, ATHandler_stub::read_string_table[ATHandler_stub::read_string_index], prefix_len)) {
-                oob->cb();
-                break;
-            }
-        }
+
+//        size_t prefix_len = 0;
+//        for (struct oob_t *oob = _oobs; oob; oob = oob->next) {
+//            prefix_len = oob->prefix_len;
+//            if (!memcmp(oob->prefix, ATHandler_stub::read_string_table[ATHandler_stub::read_string_index], prefix_len)) {
+//                oob->cb();
+//                break;
+//            }
+//        }
```

Comment:
Why is there commented out code here? If it is not required it should be removed ?

Reply:
removed